packages/ubuntu_localizations/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.4.1
+
```

Comment:
Did anything change here besides merging new translations?

Reply:
Nope!

Reply:
We should set the default title of the weblate pr:s to contain a conventional commit tag.

Reply:
Please remind me tomorrow morning :)

Reply:
Seems like we already changed that, see e.g. https://github.com/canonical/ubuntu-flutter-plugins/commit/8b180faafbb45b9c2e0686d8bc03fc2cbce42525

Reply:
Changed the prefix to `fix(l10n)` for future commits now